src/components/ContinuationIn/ContinuationAuthorization.vue

```diff
@@ -170,10 +172,28 @@ export default class ExtraproRegistration extends Mixins(DocumentMixin) {
   fileValidity = false
   customErrorMessage = ['', '', '', '', ''] // max 5 files
 
+  get minAuthorizationDate (): string {
+    // the BC Founding Date (if it exists, ie, expro business)
```

Comment:
I'm thinking we change this to a JSDoc comment. Thoughts?

Reply:
Absolutely. Now looks like this:

![image](https://github.com/bcgov/business-create-ui/assets/10002889/2b396f99-933d-4c25-9cd3-ebe4995aeece)

Reply:
Amazing. Thanks Sev! Resolving.